src/unix/stream.c

```diff
@@ -924,7 +924,6 @@ static void uv__write(uv_stream_t* stream) {
   }
 
   req->error = n;
-  // XXX(jwn): this must call uv__stream_flush_write_queue(stream, n) here, since we won't generate any more events
```

Comment:
I don't really know what this comment was trying to convey so I removed it. No comment > confusing comment.

Reply:
Done: #3668